decentraland/helpers/storage.js

```diff
@@ -8,6 +8,19 @@ const MY_ASSETS_LIST = "MY_ASSETS_LIST";
 const EPHEMERAL_ACCOUNT_CREATION_STATUS = "EPHEMERAL_ACCOUNT_CREATION_STATUS";
 const EPHEMERAL_ACCOUNT_CREATION_ACTIONS = "EPHEMERAL_ACCOUNT_CREATION_ACTIONS";
 const IS_FIRST_APP_USE = "IS_FIRST_APP_USE";
+const USER_ACTIONS = "USER_ACTIONS";
+
+export const storeUserActions = async userActions => {
+  const strUserActions = _toString(userActions);
+  await _storeData(USER_ACTIONS, strUserActions, false);
+};
+
+export const retrieveUserActions = async () => {
+  const strUserActions = await _retrieveData(USER_ACTIONS);
+  const userActions = _fromString(strUserActions);
+  if (userActions === null) return [];
```

Comment:
So we're storing `userActions` as an array rather than an object keyed by some id?

Reply:
As long as userActions and the assets list looping through both won't be a problem. If it ever becomes one, we can change one from an array to an object with a key as an "index"

Reply:
Are the "account info" actions an array or an object?

Reply:
(Not gating merging this PR)

Reply:
It is an array. But I think it should be an object.

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/342

Reply:
Cool, thanks. Resolving this.